proto/viam/app/v1/robot.proto

```diff
@@ -139,12 +139,18 @@ message NetworkConfig {
   string tls_cert_file = 3;
   string tls_key_file = 4;
   SessionsConfig sessions = 5;
+  repeated TrafficTunnelEndpoint traffic_tunnel_endpoints = 6;
 }
 
 message SessionsConfig {
   google.protobuf.Duration heartbeat_window = 1;
 }
 
+message TrafficTunnelEndpoint {
+  int32 port = 1;
+  int32 connnection_timeout_seconds = 2;
```

Comment:
Part of me wonders if this should be a `string` and be a duration in the same way as `heartbeat_window`. You didn't scope this right, @cheukt ? i.e. we didn't agree on a specific API?

Reply:
ya a `google.protobuf.Duration` makes sense

Reply:
Cool; switched and renamed `connection_timeout_seconds` -> `connection_timeout`.